pkg/serving/config_changes.go

```diff
@@ -164,7 +166,10 @@ func UpdateServiceAnnotations(service *servingv1.Service, toUpdate map[string]st
 // UpdateRevisionTemplateAnnotations updates annotations for the given Revision Template.
 // Also validates the autoscaling annotation values
 func UpdateRevisionTemplateAnnotations(template *servingv1.RevisionTemplateSpec, toUpdate map[string]string, toRemove []string) error {
-	if err := autoscaling.ValidateAnnotations(true, toUpdate); err != nil {
+	ctx := context.Background()
```

Comment:
would change here to `context.TODO()` as we don't really want to create a new context, but indicate that we just not support this when entering this library method (we could consider todo this later). Technically its the same (a new empty context is created), but semantically its a help to indicate that we are just not offering context support.

as a side note, i recently listened to https://changelog.com/gotime/143 which explained the difference nicely, too (plus a lot of other interesting information)

Reply:
now using context.TODO
thanks for link! :-)